jubatus/server/server/clustering_impl.cpp

```diff
@@ -7,7 +7,7 @@
 #include <utility>
 #include "jubatus/util/lang/shared_ptr.h"
 
-#include "../../server/framework.hpp"
+#include <jubatus/server/framework.hpp>
```

Comment:
Please use ``-i Internal include (generate #include with relative path)`` of ``jenerator`` for engines included in Jubatus source tree.   You can also do that via waf task:

```
./waf regenerate
./waf regenerate_client
```